docs/_deploying/heroku.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Deploying to Heroku
+order: 0
+slug: heroku
+lang: en
+layout: tutorial
+permalink: /deploying/heroku
+redirect_from:
+  - /deploying
+---
+# Deploying to Heroku
+
+<div class="section-content">
+This guide will walk you through preparing and deploying a Slack app using [Bolt for JavaScript](/bolt-js) and the [Heroku platform](https://heroku.com/). Along the way, we’ll download a Bolt Slack app, prepare it for Heroku, and deploy it.
+</div>
+
+When you’re finished, you’ll have this ⚡️[Getting Started with Heroku app](https://github.com/mwbrooks/bolt-js-getting-started-with-heroku-app) to run, modify, and make your own.
+
+---
+
+### Get a Bolt Slack app
+
+We'll continue from [Getting Started with Bolt for JavaScript][getting-started-guide] to learn how to prepare and deploy a Slack app to Heroku.
+
+Download the Getting Started app and change into its directory:
+
+```shell
+git clone https://github.com/mwbrooks/bolt-js-getting-started-app.git
```

Comment:
How about having this repo in `github.com/slackapi`?

Reply:
I've now transferred my repository to `github.com/slackapi` and updated the references in commit 0cad3bb 🥳